test/SlowTests/SparrowTests/LoggingSourceTests.cs

```diff
@@ -290,7 +290,9 @@ private void AssertNoFileMissing(string[] files)
                 var current = list[i];
                 if (previous.Date == current.Date && previous.Number + 1 != current.Number)
                 {
-                    if (previous.Number == current.Number && Path.GetExtension(current.FileName) == ".gz")
+                    if (previous.Number == current.Number && 
+                        (Path.GetExtension(current.FileName) == ".gz" && Path.GetExtension(current.FileName) == ".log") ||
```

Comment:
Am I missing something, but `(Path.GetExtension(current.FileName) == ".gz" && Path.GetExtension(current.FileName) == ".log")` should always be false, no?
A filename cannot have both `.gz` and `.log` extensions

Reply:
Oops. Yes, should be `previous`